app/assets/stylesheets/components/_container.sass

```diff
@@ -1,5 +1,7 @@
 $Container-maxWidth: 1440px
+$Container-spacing: 2 * $Theme-spacing-base
 
 .Container
   margin: 0 auto
   max-width: $Container-maxWidth
+  padding-left: $Container-spacing
```

Comment:
I don't recall what was the reason, but with this padding left the container is not centered.